Foundation/NSDictionary.swift

```diff
@@ -278,7 +278,19 @@ public class NSDictionary : NSObject, NSCopying, NSMutableCopying, NSSecureCodin
     public func descriptionWithLocale(locale: AnyObject?, indent level: Int) -> String { NSUnimplemented() }
     
     public func isEqualToDictionary(otherDictionary: [NSObject : AnyObject]) -> Bool {
-        NSUnimplemented()
+        if count != otherDictionary.count {
+            return false
+        }
+        
+        for (key, value) in _storage {
```

Comment:
Is NSDictionary subclassable?  Then iterating over _storage wouldn't use the data from the subclass.